sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
 
 ### Other Changes
 
+* Update OpenTelemetry dependencies.
```

Comment:
Why should we maintain in two places? This causes a confusion. I recommend only keeping 1.1.1 changes.

Reply:
If sorting by release version, the 1.2.0 beta 1, 2, and 3 don't have the hotfix change so I think this entry in the changelog is valid.
If reading the changelog top-down, the previous "update dependencies" entry specifies different versions.

I can remove this section if you would prefer, but I don't think it's something that we would have to maintain.